helm/configs/backend-next/development/values.yaml

```diff
@@ -3,7 +3,23 @@ envValue: development
 host: scicat.development.psi.ch
 
 ingress:
-  annotations: 
+  annotations:
     b64/nginx.ingress.kubernetes.io/whitelist-source-range: "{{ .Values.secretsJson.OPENEM_WHITELISTED_IPS }}"
 
 migrateCommand: npm run migrate:db:up
+
+# Run on nodes whitelisted by scopem
+# Can be removed when scopem-openem.ethz.ch is publically accessible
+affinity:
```

Comment:
would it be easier to simply use

```yaml
  nodeSelector:
    workerset: dev-pool1
```
?

inplace of `affinity`

Reply:
This is clean, but isn't it better to list all nodes?